### ISSUE TITLE
Implement the `train_models()` function in `src/train_model.py`

### ISSUE THREAD

**Issue by cachebag** [cachebag/BE-2100-Final-Project#2]
### Task
Implement the `train_models()` function in `src/train_model.py`

Make sure Ali has finished the `load_data()` function before you start testing.

### What you need to do
Your function needs to:
1. Train 3 different models: Logistic Regression, Random Forest, and SVM
2. Calculate training and validation accuracy for each model
3. Store the results in a dictionary and return it

### Dictionary structure to return
```python
results[model_name] = {
    'model': model_object,
    'train_acc': training_accuracy,
    'val_acc': validation_accuracy
}
```

### Steps
- Create the 3 model objects (imports are already at the top of the file)
- For Logistic Regression: use `max_iter=1000, random_state=42`
- For Random Forest: use `n_estimators=100, random_state=42`
- For SVM: use `kernel='rbf', random_state=42`
- Use `.fit()` to train each model
- Use `.predict()` to get predictions
- Use `accuracy_score()` from sklearn to calculate accuracies
- Print the accuracies as you go so you can see the results

### How to test
Run the `train_model.py` script from the `src/` directory. You should see training progress and accuracy scores printed out for each model.
